src/components/Json-results.js

```diff
@@ -1,38 +1,35 @@
 import React from 'react';
 import Intent from '../intent/json-intent';
 
-var JsonResults = React.createClass({
+export default class extends React.Component {
```

Comment:
I'll be keeping away from the Class in ES6

https://medium.com/javascript-scene/how-to-fix-the-es6-class-keyword-2d42bb3f4caf